code/modules/jobs/job_types/_job.dm

```diff
@@ -250,7 +250,7 @@
 /datum/outfit/job
 	name = "Standard Gear"
 
-	var/jobtype = null
+	var/jobtype = /datum/job/assistant //defaults to assistant to prevent ID nonesense
```

Comment:
this seems like a bad idea since it wont break if a job isnt set

Reply:
Thats kinda the point. Also this is just to fix the runtime when the outfit is set to datum/outfit/job.

Reply:
In what case is that intentional?